src/array.c

```diff
@@ -801,7 +801,7 @@ STATIC_INLINE void jl_array_grow_at_beg(jl_array_t *a, size_t idx, size_t inc,
     size_t nbinc = inc * elsz;
     char *data = (char*)a->data;
     char *newdata;
-    char *typetagdata;
+    char *typetagdata = NULL;
```

Comment:
how is this change related?

Reply:
It is completely unrelated -- just fixes a warning while building Julia. I can drop it from this PR...?